modules/angular-meteor-meteorCollection.js

```diff
@@ -239,48 +248,25 @@ angularMeteorCollections.factory('$meteorCollection', ['AngularMeteorCollection'
       }
 
       var ngCollection = new AngularMeteorCollection(reactiveFunc(), collection);
-      var realOldItems;
 
       function setAutoBind() {
         if (auto) { // Deep watches the model and performs autobind.
           ngCollection.unregisterAutoBind = $rootScope.$watch(function () {
-            if (ngCollection.UPDATING_FROM_SERVER){
-              realOldItems = _.without(ngCollection, 'UPDATING_FROM_SERVER');
+            if (ngCollection.UPDATING_FROM_SERVER) {
               return 'UPDATING_FROM_SERVER';
             }
             return _.without(ngCollection, 'UPDATING_FROM_SERVER');
           }, function (newItems, oldItems) {
-            if (newItems == 'UPDATING_FROM_SERVER')
+            if (newItems == 'UPDATING_FROM_SERVER' ||
```

Comment:
But what happens if the oldItems is 'UPDATING_FROM_SERVER' and the new items are new collection?
it means we got a change from the client and here you are immediately return without updating the server...